Maple2.Server.DebugGame/Graphics/DebugFieldWindow.cs

```diff
@@ -9,18 +10,21 @@
 namespace Maple2.Server.DebugGame.Graphics;
 
 public class DebugFieldWindow {
-    public DebugGraphicsContext Context { get; init; }
+    private DebugGraphicsContext Context { get; }
     public DebugFieldRenderer? ActiveRenderer { get; private set; }
     public IWindow? DebuggerWindow { get; private set; }
-    public IInputContext? Input { get; private set; }
-    public ComPtr<IDXGISwapChain1> DxSwapChain { get; private set; }
-    public ImGuiController? ImGuiController { get; private set; }
-    public int WindowId { get; init; }
-    public string WindowName { get; init; }
+    private IInputContext? Input { get; set; }
+    private ComPtr<IDXGISwapChain1> DxSwapChain { get; set; }
+    private ImGuiController? ImGuiController { get; set; }
+    private int WindowId { get; }
```

Comment:
Why does this need to be private?

Reply:
It doesn't need to be but it was just editor complaining